.github/workflows/main.yml

```diff
@@ -100,7 +100,8 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        wallet: ["vdrtools_wallet", "askar_wallet"]
+        main_wallet: ["main_vdrtools_wallet,vdrtools_wallet", "main_askar_wallet,askar_wallet"]
```

Comment:
Let's have a separate job for migration clippy check. This seems like slippery slope if we were to start combining with even more feature flag going forward. 

In fact, let's keep just `["vdrtools_wallet", "askar_wallet"]` here when clippy-ing workspace holistically.